org.eclipse.winery.topologymodeler.ui/src/app/canvas/canvas.component.ts

```diff
@@ -1018,6 +1048,53 @@ export class CanvasComponent implements OnInit, OnDestroy, OnChanges, AfterViewI
         this.importTopologyData.selectedTopologyTemplateId = null;
     }
 
+    /**
+     * Closes the threat modeling modal
+     */
+    closeThreatModeling(): void {
+        this.threatModelingModal.hide();
+    }
+
+    /**
+     *
+     *
+     */
+    createNewThreat(): void {
+        console.log(this.threatModelingData.threatCreation);
```

Comment:
Please get rid of all your `console.log()`s

Reply:
fixed